packages/core/lib/src/data/build_bits.dart

```diff
@@ -81,7 +81,7 @@ abstract class BuildBit<InputType, OutputType> {
   /// Returns `null` to use configuration from the previous bit.
   ///
   /// By default, do swallow if not [isInline].
-  bool get swallowWhitespace => !isInline;
+  bool? get swallowWhitespace => !isInline;
```

Comment:
```suggestion
  bool get swallowWhitespace => !isInline;
```

Reply:
`_TagABit` overrides swallowWhitespace with null, so I don't think this is possible

Reply:
Ah, that behavior is noted in the dartdoc. Sorry for this 😅